files/image_config/config-chassis/config-chassis.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+# 
+###########################################################################
```

Comment:
Revisiting this PR again, have a question - if we really need this additional service config-chassis.service to configure the chassis? Seems like we are binding starting of http service with chassis.  

Can we make it as a generic http service and have control variables in for eg: in platform_env.conf to start stop service?